### ISSUE TITLE
Cannot login to the admin GUI after install.

### ISSUE THREAD

**Issue by louismunro** [inverse-inc/packetfence#1000]
Clean install.
Went through the configurator. 

Password is not being accepted at /admin. 
The logs seem to indicate that the auth sources keep processing after a successful match  on the database:

```
Nov 05 14:31:43 httpd.admin(2713) INFO: Authentication successful for admin in source local (SQL) (pf::authentication::authenticate)
Nov 05 14:31:48 httpd.admin(2711) INFO: Password validation failed for admin: passwords don't match (pf::password::validate_password)
Nov 05 14:31:48 httpd.admin(2711) ERROR: unable to read password file '/usr/local/pf/conf/admin.conf' (pf::Authentication::Source::HtpasswdSource::authenticate)
Nov 05 14:31:48 httpd.admin(2711) ERROR: unable to read password file '/usr/local/pf/conf/admin.conf' (pf::Authentication::Source::HtpasswdSource::authenticate)
(END) 
```

**Comment by julsemaan** [comment]
This happened to me also on a fresh install

**Comment by nirmal4wifi** [comment]
You may have to check system date. If system date is incorrect you wont be able to login.
Hope this helps.

**Comment by louismunro** [comment]
No. It's a bug with the Apache ChildInithandler.
The Statsd object goes out of scope somehow on initial install.
That causes an exception that is caught in Catalyst and results in an HTTP 500.

Don't know yet why the object should go out of scope (or be undefed).

**Comment by jrouzierinverse** [comment]
Fixed in commit 73742c1900a95f9ca23a175a20b425eb01d4cfeb
